packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -149,7 +149,7 @@ export class SceneControllerConfigurationCCAPI extends CCAPI {
 				await this.set(propertyKey, value, dimmingDuration);
 			}
 		} else if (property === "dimmingDuration") {
-			if (typeof value !== "number") {
+			if (typeof value !== "string" && !Duration.isDuration(value)) {
```

Comment:
```suggestion
			if (typeof value !== "string" && !(value instanceof Duration)) {
```
We currently don't accept arbitrary objects that look like a Duration. If we were to change that, it needs to be part of another PR that changes all locations. The `Duration.isDuration` method itself is also not necessary here.